app.rb

```diff
@@ -0,0 +1,53 @@
+require './student'
+require './teacher'
+require './book'
+require './rental'
+require './list_items'
```

Comment:
- [ ] When I run your app It's giving an error because here it should be `'./list_item'`  instead of `'./list_items'`, Because your file name is `list_item`. Please fix this issue.

Reply:
thanks very much for this review i have implemented the required changes